README.adoc

```diff
@@ -61,6 +61,18 @@ assert_equal(nil, actual)
 assert_nil(actual)
 ----
 
+=== Assert Truthy [[assert-truthy]]
+
+Use `assert` if expecting `true` value instead of `assert_equal`.
```

Comment:
Probably we should make this more generic and add some `false` example as well.

Reply:
I did not understood what you meant by false examples in assert truth but if you meant to add assertion to test false values. Minitest uses `refute`. That can be covered in another PR I think.

Reply:
Yeah, I meant we should add something about `refute`, but it can be a different rule.